src/commands/scanner/rule/add.ts

```diff
@@ -0,0 +1,78 @@
+import { flags, SfdxCommand } from '@salesforce/command';
+import { Messages, SfdxError } from '@salesforce/core';
+import { CustomRuleRegistrar } from '../../../lib/pmd/CustomRuleRegistrar';
+
+
+// Initialize Messages with the current plugin directory
+Messages.importMessagesDirectory(__dirname);
+
+// Load the specific messages for this file. Messages from @salesforce/command, @salesforce/core,
+// or any library that is using the messages framework can also be loaded this way.
+const messages = Messages.loadMessages('scanner', 'add');
+
+
+export default class Addcustom extends SfdxCommand {
+
+  public static description = messages.getMessage('commandDescription');
+
+  public static examples = [
+    `$ sfdx scanner:rule:add --language "apex" --path "/dir/to/jar/lib"
+        (todo: add sample output here)
+        
+        $ sfdx scanner:rule:add --language "apex" --path "/file/path/to/customrule.jar,/dir/to/jar/lib"
+        (todo: add sample output here)
+        `
+  ];
+
+  protected static flagsConfig = {
+    language: flags.string({ 
+        char: 'l', 
+        description: messages.getMessage('languageFlagDescription'),
+        required: true
+    }),
+    path: flags.string({ 
```

Comment:
This should also be an array, so you can specify multiple paths at once. Also, why not give it a `char` of `p`?

Reply:
Multiple paths are taken as a comma separated value.

Reply:
Will add -p